src/amp-story-player/amp-story-player-impl.js

```diff
@@ -793,11 +818,13 @@ export class AmpStoryPlayer {
    * @param {!Element} story
    * @param {!Element} iframe
    * @param {!VisibilityState} visibilityState
+   * @param {EmbedMode=} embedMode
    * @private
    */
-  layoutIframe_(story, iframe, visibilityState) {
+  layoutIframe_(story, iframe, visibilityState, embedMode) {
```

Comment:
If the param is optional you should specify a default value

```suggestion
  layoutIframe_(story, iframe, visibilityState, embedMode = undefined) {
```